cli/options/src/lib.rs

```diff
@@ -259,10 +259,11 @@ pub struct TranslationOptions {
     /// prefixed by modifiers are processed from left to right,
     /// excluding or including items. Each pattern selects a number of
     /// item. The modifiers are:
-    ///
-    ///  - `+`: includes the selected items with their dependencies,
-    /// transitively (e.g. if function `f` calls `g` which in turn
-    /// calls `h`, then `+k::f` includes `f`, `g` and `h`)
+
+    /// {n}{n} - `+`: includes the selected items with their
```

Comment:
I still don't understand why you don't just use regular comments that produce the spaces for you.

Reply:
That's because I want `clap` (or `rustdoc` at some other places) to look nice! In this situation, one newline is ignored by `clap` and the usual double space trick for markdown is not parsed. Here I could use one empty line, but then the spacing of the dashes are not aligned (clap seems to trim lines in a special way when they are prefixed with `{n}`)